tests/Aspire.Hosting.Testing.Tests/TestingFactoryTests.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Aspire.Hosting.Testing.Tests;
 
-public class TestingFactoryTests(DistributedApplicationFixture<Program> fixture) : IClassFixture<DistributedApplicationFixture<Program>>
+public class TestingFactoryTests(DistributedApplicationFixture<Projects.TestingAppHost1_AppHost> fixture) : IClassFixture<DistributedApplicationFixture<Projects.TestingAppHost1_AppHost>>
```

Comment:
Is this type being generated into the AppHost project or the test project?

Reply:
Yeah it was already there, @davidfowl added it a while back and I just updated it to be public.